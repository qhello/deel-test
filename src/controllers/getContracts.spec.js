const request = require("supertest");
const app = require("../app");

describe("getContracts", () => {
  it("should return contracts", async () => {
    const res = await request(app).get("/contracts").set("profile_id", 1);

    expect(res.statusCode).toEqual(200);
    expect(res.body).toMatchSnapshot([
      {
        createdAt: expect.any(String),
        updatedAt: expect.any(String),
      },
    ]);
  });
});
