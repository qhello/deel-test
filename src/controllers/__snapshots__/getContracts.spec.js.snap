// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getContracts should return contracts 1`] = `
[
  {
    "ClientId": 1,
    "ContractorId": 6,
    "createdAt": "2022-09-27T11:11:32.728Z",
    "id": 2,
    "status": "in_progress",
    "terms": "bla bla bla",
    "updatedAt": "2022-09-27T11:11:32.728Z",
  },
]
`;
